src/data/moves/move-attrs/suppress-abilities-attr.ts

```diff
@@ -30,4 +32,16 @@ export class SuppressAbilitiesAttr extends MoveEffectAttr {
     return (_user, target, _move) =>
       !target.getAbility().hasAttrFlag(AbAttrFlag.UNSUPPRESSABLE_ABILITY) && !target.summonData.abilitySuppressed;
   }
+
+  /**
+   * If the target has a {@link HIGH_VALUE_ABILITIES | high-value ability}, grants (+2)
+   * effect score. Otherwise, this has a 60% chance to grant (+1).
+   */
+  override getEffectScore(user: EnemyPokemon, target: Pokemon, _move: Move): number {
+    const targetHasHighValueAbility = target
+      .getAbilities({ revealedOnly: true })
+      .some((ab) => !ab.passive && HIGH_VALUE_ABILITIES.includes(ab.ability.id));
+
+    return targetHasHighValueAbility ? 2 : this.getRandomScore(user, 60);
```

Comment:
These `+2` and `60%` numbers should probably be descriptive constant values

and not a _magic numbers_